packages/core/src/result-transformers.ts

```diff
@@ -52,7 +52,8 @@ class ResultTransformers {
    * const { keys, records, summary } = await driver.executeQuery('CREATE (p:Person{ name: $name }) RETURN p', { name: 'Person1'})
    *
    * @returns {ResultTransformer<EagerResult<Entries>>} The result transformer
-   * @alias {@link ResultTransformers#eager}
+   * @deprecated This is deprecated, use {@link ResultTransformers#eager} instead
```

Comment:
Might be worth mentioning that `#eager` is a drop-in replacement/alias/exactly the same.

Similarly for `#mapped`, won't comment again.